CHANGELOG.rst

```diff
@@ -63,6 +63,7 @@ Changed
 - ``utter_custom_message()`` method in rasa_core_sdk has been renamed to ``utter_elements()``
 - updated dependencies. as part of this, models for spacy need to be reinstalled
   for 2.1 (from 2.0)
+- renamed ``rasa show stories`` to ``rasa visualize``
```

Comment:
don't think this is necessary as CLI isn't released yet

Reply:
added an entry for every update I did, so will keep it

Reply:
Hmm I think @MetcalfeTom might have deleted some actually https://github.com/RasaHQ/rasa/pull/3366/files

Reply:
I can still see some entries. So what do you propose, delete them all?

Reply:
Yeah I guess so, since "changed" is more about what's changed since the last version. Tom might have clearer ideas for what the changelog should look like though. I.e. maybe `add --fixed-model-name to rasa train to specify name of stored model` should actually be something like `add --fixed-model-name to train command to specify name of stored model` in the vein of `removed --num_threads from run command (server will be asynchronous but running in a single thread)`

Reply:
There's only a couple entries to remove here.  `--fixed-model-name` was already possible, so we don't need to add a note saying we added it (because we also removed it since `rasa_nlu==0.15`)

As for combined training, whatever is implemented now will be default behaviour since this wasn't possible before, so no `changes` there